static/src/stylesheets/module/content-garnett/_types.scss

```diff
@@ -934,8 +935,23 @@ $quote-mark: 35px;
         }
 
     }
+    .content__headline__byline {
+        overflow: visible;
+    }
+
     .meta__image {
         float: none;
+        margin-right: -$gs-gutter;
+        @include mq(phablet) {
+            margin-right: 0;
+        }
+    }
+
+    .content__head--byline-pic .byline-img {
+
+        @include mq(mobileLandscape) {
+            height: gs-span(2.5);
```

Comment:
😮

Reply:
haha, I'm referencing how it's already written in _content.scss. But I suppose two wrongs don't make a right, will update!